### DISCUSSION TITLE
如何在 Lite Mode 下设置超过5个节点

### DISCUSSION THREAD

**Discussion by LightYearZh** [opengnb/opengnb#19] [Q&A]
用户手册里提到可以在 Lite Mode下设置更多的节点，没具体说明怎么设。用1006以后的nodeid启动提示local_node is NULL。怎么添加节点？涉及到的 -a -r 两个参数的传参格式是什么样的

**Comment by gnbdev** [comment]
以后会完善这里的文档。
现在，可以看代码里的实现。

**Comment by humbass** [comment]
> 用户手册里提到可以在 Lite Mode下设置更多的节点，没具体说明怎么设。用1006以后的nodeid启动提示local_node is NULL。怎么添加节点？涉及到的 -a -r 两个参数的传参格式是什么样的

跟你一样没成功使用 Lite Mode 超过 6 台，还是用 Save Mode 来配置了.
